backend/Dockerfile

```diff
@@ -23,4 +23,6 @@ RUN sed -i 's/bind 127.0.0.1/bind 0.0.0.0/g' /etc/redis.conf \
     # backwards compatibility
     && ln -s /usr/local/lib/pyenv/versions/3.6.8/bin/python /usr/local/bin
 
+COPY ./supervisor.d/* /etc/supervisor/conf.d/
```

Comment:
why is this required? Is the default location changed?

Reply:
they weren't default configuration so a volume was necessary. I added them on the image so if someone runs the image by itself without mounting it should work

Reply:
I do not understand, do current users need to change sth in their volume mapping for supervisor confs? Same for monitor. If yes, we need to document it and add new upgrade instructions.

Reply:
no. if you run the image without a volume it doesn't have a default supervisor configuration.